apis/python/src/tiledbsoma/util_tiledb.py

```diff
@@ -139,3 +139,26 @@ def list_fragments(array_uri: str) -> None:
 
     frags_df = pd.DataFrame(fragments)
     print(frags_df)
+
+
+def split_column_names(
+    array_schema: tiledb.ArraySchema, column_names: Optional[Sequence[str]]
+) -> Tuple[Union[Sequence[str], None], Union[Sequence[str], None]]:
+    """
+    Given a tiledb ArraySchema and a list of dim or column names, split
```

Comment:
Given a tiledb ArraySchema and a list of dim or attr names, split
them into a tuple of (dim_names, attr_names).

Idea: inputs are "column names"; outputs are "dim names" and "attr names"

Reply:
fixed & expanded docstring